endpoint.go

```diff
@@ -400,6 +400,12 @@ func (ep *endpoint) sbJoin(sbox Sandbox, options ...EndpointOption) error {
 		}
 	}()
 
+	sb.config.dnsList = append(sb.config.dnsList, ep.iface.dnsServers...)
+	sb.config.dnsSearchList = append(sb.config.dnsSearchList, ep.iface.dnsSearchDomains...)
+	if err = sb.setupResolutionFiles(); err != nil {
```

Comment:
am not quite sure if we have to call `setupResolutionFiles` every time from `sbJoin`.
Instead, I would suggest to make the change in `Refresh()` in sandbox.go to call it just the once after all the 
endpoint Joins are processed.

@sanimej is the best person to give an authoritative answer to that.